CHANGELOG.md

```diff
@@ -8,12 +8,66 @@ Subheadings to categorize changes are `added, changed, deprecated, removed, fixe
 
 # Changelog
 
-The latest published Parley release is [0.6.0](#060---2025-10-06) which was released on 2025-10-06.
-You can find its changes [documented below](#060---2025-10-06).
+The latest published Parley release is [0.7.0](#060---2025-11-24) which was released on 2025-11-24.
+You can find its changes [documented below](#060---2025-11-24).
 
 ## [Unreleased]
 
-This release has an [MSRV] of 1.82.
+This release has an [MSRV] of 1.83.
+
+## [0.7.0] - 2025-11-24
+
+This release has an [MSRV] of 1.83.
+
+### Highlights
+
+([#448][] by [@taj-p][]) and ([#449][] by [@nicoburns][]) collectively fix a significant performance bug that occurred when laying
+out large paragraphs of text. Previously the time to perform layout was non-linear with respect to the input size and laying out
+paragraphs of text with more than ~1k characters was very slow.
+
+The new `TextWrapMode` style implements the semantics of the [`text-wrap-mode`](https://developer.mozilla.org/en-US/docs/Web/CSS/Reference/Properties/text-wrap-mode)
+CSS property and allows text-wrapping to be disabled completely for a span of text.
+
+
+### Migration
+
+Some modules have been moved:
+
+- `parley::editor` and `parley::layout::editor` are now `parley::editing`
+- `parley::layout::cursor` is now `parley::cursor`
+
+Fontique no longer sets the `dlopen` feature of `yeslogic-fontconfig-sys` by default. If you wish to run Fontique on a Linux system without fontconfig
+installed then you will need to enable the new `dlopen` feature of the `fontique` crate.
+
+### Added
+
+#### Parley
+
+- Add `TextWrapMode` style. This allow line wrapping to be disabled completely for a span of text (excluding explicit line breaks) ([#367][] by [@nicoburns][])
+- Add `Cluster::from_point_exact` method for hit-testing spans of text. This is useful for implementing "hover" or "click" functionality. ([#447][] by [@nicoburns][])
+
+### Changed
+
+#### Parley
+- Split off various modules into "editing" folder. ([#440][] by [@PoignardAzur][])
+- Split contents of layout/mod.rs file. ([#444][] by [@PoignardAzur][])
```

Comment:
Olivier's link reference definition also needs to be added. This is part of why we recommend running markdownlint as part of the release process.